packetery/src/Packetery/Module/Product/DataTab.php

```diff
@@ -85,7 +85,7 @@ public function registerTab( array $tabs ): array {
 	 * @return Form
 	 */
 	private function createForm( Product\Entity $product ): Form {
-		$form = $this->formFactory->createForm();
```

Comment:
Pokud už createForm není potřeba, prosím odstranit.

Reply:
$this->formFactory->createForm() je součástí Nette v packetery_vendoru a na to se šahat nesmí.

Reply:
Máš pravdu, promiň.